### PR TITLE
fix: set endpoint and instance selectors with url parameters

### PR DESCRIPTION
**Pull request by Fine0830**
Fixes endpoint and instance selectors with url parameters.

video 

https://user-images.githubusercontent.com/20871783/235704561-4a413be8-07cb-4e16-b4c6-03ed0a2b3d32.mov

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/views/dashboard/panel/Tool.vue [hunk 1]
```diff
@@ -602,13 +619,29 @@ limitations under the License. -->
...
   }
-  function updateCurrentPod() {
+  async function updateCurrentPod(type: string) {
+    if (params.podId) {
+      let resp;
+      if (type === EntityType[2].value) {
+        resp = await selectorStore.getEndpoint(params.podId);
+      } else {
+        resp = await selectorStore.getInstance(params.podId);
+      }
```
[the remark this answers is not in the document]
Comment (wu-sheng) [reply src/views/dashboard/panel/Tool.vue:629]: Also, `listInstances` always provides the full list. `findEndpoint` is not reliable in the other hand.

[the remark this answers is not in the document]
Comment (wu-sheng) [reply src/views/dashboard/panel/Tool.vue:629]: We should have a pop-out box saying `instance`/`endpoint` ID doesn't exist.

File: src/views/dashboard/panel/Tool.vue [hunk 2]
```diff
@@ -585,6 +604,19 @@ limitations under the License. -->
...
   }
+  async function updateCurrentDestEndpoint() {
+    const resp = await selectorStore.getEndpoint(params.destPodId, true);
+
+    if (resp.errors) {
+      return ElMessage.error(resp.errors);
+    }
+    const pod = resp.data.endpoint || {};
+    if (!pod.id) {
+      ElMessage.info("The Destination endpoint ID doesn't exist.");
```
Comment (wu-sheng) [inline src/views/dashboard/panel/Tool.vue:615]: `Destination`? Is this related to Endpoint Relation?

Comment (wu-sheng) [reply src/views/dashboard/panel/Tool.vue:615]: How about source?

Comment (Fine0830) [reply src/views/dashboard/panel/Tool.vue:615]: > `Destination`? Is this related to Endpoint Relation?

Yes

Comment (Fine0830) [reply src/views/dashboard/panel/Tool.vue:615]: > How about source?

it is named `endpoint` or `instance`.

**Review by wu-sheng — APPROVED** [review]
